owncloud/owncloud.py

```diff
@@ -804,6 +1022,105 @@ def delete_attribute(self, app, key):
             return True
         raise ResponseError(res)
 
+
+    def get_apps(self):
+    	""" List all enabled apps through the provisioning api.
+	    If this fails, try to run enable_app_ajax('provisioning_api') first.
+	    Returns a dict of apps, with values True/False, representing the enabled state.
+	"""
+	ena_apps = {}
```

Comment:
I'll figure out what needs to be updated.  I'm new to python and need to 
learn the ins and outs.

On 3/11/2015 11:02 AM, Christopher Schäpers wrote:
>
> In owncloud/owncloud.py 
> <https://github.com/owncloud/pyocclient/pull/76#discussion_r26219408>:
>
> > @@ -804,6 +1022,105 @@ def delete_attribute(self, app, key):
> >              return True
> >          raise ResponseError(res)
> >
> > +
> > +    def get_apps(self):
> > +    	""" List all enabled apps through the provisioning api.
> > +	    If this fails, try to run enable_app_ajax('provisioning_api') first.
> > +	    Returns a dict of apps, with values True/False, representing the enabled state.
> > +	"""
> > +	ena_apps = {}
>
> Yeah, that's why PyCharm went totally crazy with errors – 90% of it 
> were PEP8 stuff though. Should I take a shot a making this project 
> PEP8 compatible?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/owncloud/pyocclient/pull/76/files#r26219408>.
>